live-examples/css-examples/animation/animation-direction.css

```diff
@@ -0,0 +1,47 @@
+#example-element {
+    background-color: #1766aa;
+    color: white;
+    margin: auto;
+    margin-left: 0;
+    border: 5px solid #333;
+    width: 150px;
+    height: 150px;
+    border-radius: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
```

Comment:
I'm not sure you need lines 10-13 for this or the other new examples. I might be wrong though (I know you do for animation-delay, because that has extra bits. And I think maybe you'll need to add extra bits to these examples as well.